lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -40,6 +40,15 @@ function subscribeEvents() {
     const label = args !== null ? '#' + args.attr('id') : '';
     ga('send', 'event', 'Modal', 'Close', label);
   });
+
+  // Signup button on pitch page
+  const isPitchPage = $('.pitch').length > 0;
+  if (isPitchPage) {
+    // This applies to both authenticated & anonymous users. It also covers both modal register & login.
+    $('.header__signup form, #user-login-form, #user-register-form').on('submit', () => {
```

Comment:
@DFurnes Actually looks like I didn't catch it the first time, I think I'll need to do the Validation Submitted approach